### PR TITLE
Update 2s-data-practice-draft-1.Rmd

### PR DESCRIPTION
**Pull request by cfcraig**
minor change to jackson's rmd script

### CODE REVIEW

**Review by hughej3 — CHANGES_REQUESTED** [review]
wtf

File: processing-and-analysis/02-command-files/2s-practice-jackson/2s-data-practice-draft-1.Rmd [hunk 1]
```diff
@@ -7,6 +7,8 @@ output:
     keep_md: yes
 ---
 
+MINOR CHANGE TO JH'S SCRIPT
```
Comment (hughej3) [inline processing-and-analysis/02-command-files/2s-practice-jackson/2s-data-practice-draft-1.Rmd:10]: why

Comment (cfcraig) [reply processing-and-analysis/02-command-files/2s-practice-jackson/2s-data-practice-draft-1.Rmd:10]: because!

**Review by hughej3 — APPROVED** [review]
thanks!

**Pull request merged**